Permissions.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+import { NativeModules, PermissionsAndroid, Platform } from 'react-native';
+
+const { WebRTCModule } = NativeModules;
+
+/**
+ * Class implementing a subset of W3C's Permissions API as defined by:
+ * https://www.w3.org/TR/permissions/
+ */
+class Permissions {
```

Comment:
In the future I think it makes sense to add some `registerPermissionHandler` so other perissions such as location or calendar can be added from the outside. And maybe even break this out to a separate package. It's good for now though.